src/main/java/com/cloudbees/jenkins/GitHubRepositoryNameContributor.java

```diff
@@ -46,7 +76,7 @@
 
 
     static abstract class AbstractFromSCMImpl extends GitHubRepositoryNameContributor {
```

Comment:
BTW since this now has only one subclass, the code could be simplified a bit by collapsing the two classes into one.